packages/nx/src/command-line/report/report.ts

```diff
@@ -77,6 +78,14 @@ export async function reportHandler() {
     );
   });
 
+  if (crystalPlugins.length) {
+    bodyLines.push(LINE_SEPARATOR);
+    bodyLines.push('Nx Crystal Plugins:');
```

Comment:
Please change this to `Registered Plugins`.

Crystal is just a codename, it shouldn't be shown in output.